spec/shared_examples.rb

```diff
@@ -29,6 +29,19 @@
     end
   end
 
+  context 'Prepare' do
+    it 'should prepare resource' do
+      res = @manager.prepare(@resource['id'])
+      expect(res.instance_variable_get(:@entity)).to be_nil
```

Comment:
I understand but this test whether :@entity is nil or not -> @entity contains hash. I cannot test it other way, because when I access res.entity it will create request and load attributes to :@entity